packages/moonstone/ExpandableItem/ExpandableItem.less

```diff
@@ -24,9 +24,9 @@
 	&.open {
 		.labeledItem {
 			&.auto {
-				.label {
-					display: none;
-				}
+					.label {
```

Comment:
Based on the GH preview, these changes appear to be adding extra tabs and should be reverted